content/news/028/index.md

```diff
@@ -66,6 +66,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [BENDYWORM]
+
+![An animation of a platformer where the entire world bends around
+you](bendyworm.gif) _Watch out! The entire world bends around you!_
+
+BENDYWORM ([GitHub][BENDYWORM-github],
+[Twitter](https://twitter.com/bauxitedev/status/1466034866122891266), by
+[@bauxitedev](https://twitter.com/bauxitedev)) is a platformer with a twist:
+the entire world bends and twists around you as your progress through the
+level. Why? Because you're inside of a gigantic worm, and worms are bendy.
```

Comment:
Can you extract all links to the bottom? I see that some are, but not all.

Example: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/860/files